R/rocm_smi_lib/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "rocm_smi_lib"
+version = v"4.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/RadeonOpenCompute/rocm_smi_lib/archive/refs/tags/rocm-$(version).tar.gz", "93d19229b5a511021bf836ddc2a9922e744bf8ee52ee0e2829645064301320f4")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/rocm_smi_lib-rocm-*/
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_PREFIX_PATH=${prefix} ..
```

Comment:
You should use the toolchain file https://docs.binarybuilder.org/stable/build_tips/#CMake-builds

Reply:
oh, noted! I just copied this from `rocminfo_jll`. I will have to look into what these differences mean, as I do not have experience with cmake.